doc/api/stream.md

```diff
@@ -1555,14 +1555,27 @@ const cleanup = finished(rs, (err) => {
 });
 ```
 
-### `stream.pipeline(...streams, callback)`
+### `stream.pipeline(src, ...transforms, dst, callback)`
```

Comment:
I would avoid the abbreviations here. s/src/source and s/dst/destination, or s/src/readable and s/dst/writable